Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -232,24 +232,172 @@ function Get-ServiceFabricApplicationAction
     return Get-ServiceFabricApplication @getApplicationParams
 }
 
+function Get-ServiceFabricApplicationUpgradeAction
+{
+    Param (
+        [string]
+        $ApplicationName
+    )
+
+    $global:operationId = $SF_Operations.GetApplicationUpgradeStatus
+    return Get-ServiceFabricApplicationUpgrade -ApplicationName $ApplicationName
```

Comment:
No retry here?

Reply:
This function has not changed. Github is not showing diff properly.